File: common/src/main/java/org/sandboxpowered/silica/network/play/clientbound/DeclareRecipes.java
package org.sandboxpowered.silica.network.play.clientbound;

import org.sandboxpowered.silica.network.*;

public class DeclareRecipes implements PacketPlay {
    @Override
    public void read(PacketByteBuf buf) {

    }

    @Override
    public void write(PacketByteBuf buf) {
        buf.writeVarInt(0);
    }

    @Override
    public void handle(PacketHandler packetHandler, PlayContext context) {

    }
}

File: common/src/main/kotlin/org/sandboxpowered/silica/network/play/clientbound/DeclareRecipes.kt
package org.sandboxpowered.silica.network.play.clientbound

import org.sandboxpowered.silica.network.PacketByteBuf
import org.sandboxpowered.silica.network.PacketHandler
import org.sandboxpowered.silica.network.PacketPlay
import org.sandboxpowered.silica.network.PlayContext

class DeclareRecipes : PacketPlay {
    override fun read(buf: PacketByteBuf) {}
    override fun write(buf: PacketByteBuf) {
        buf.writeVarInt(0)
    }

    override fun handle(packetHandler: PacketHandler, context: PlayContext) {}
}